Sources/PaymentsCore/Networking/AnalyticsService.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+class AnalyticsService {
+
+    // MARK: - Internal Properties
+
+    private let http: HTTP
+    let sessionID = UUID().uuidString.replacingOccurrences(of: "-", with: "")
+
+    // MARK: - Initializer
+
+    init(http: HTTP) {
+        self.http = http
+    }
+    
+    // MARK: - Internal Methods
+
+    func sendEvent(_ name: String) async {
+        let eventData = AnalyticsEventData(eventName: name, sessionID: sessionID)
+
+        do {
+            let analyticsEventRequest = try AnalyticsEventRequest(eventData: eventData)
+            let (_) = try await http.performRequest(endpoint: analyticsEventRequest)
+        } catch let error {
```

Comment:
I believe catch provides a local error property automatically so we should just be able to do this without adding `let error` after `catch`:
```
        } catch {
            NSLog("[PayPal SDK] Failed to send analytics: %@", error.localizedDescription)
        }
```